### PR TITLE
docs/control-service: update contributor documentation

### PR DESCRIPTION
**Pull request by DeltaMichael**
## Why?

Contributor doc for control-service is inaccurate and hard to use

## What?

Restructure document so how-to-run sections are at the beginning Update running locally section with correct instructions

## How has this been tested?

Ran the service locally using updated instructions

## What type of change are you making?

Documentation change, non-breaking

## Follow-up tickets

https://github.com/vmware/versatile-data-kit/issues/1780

### CODE REVIEW

File: projects/control-service/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,131 @@
...
-./gradlew :pipelines_control_service:bootRun
+
+Versatile Data Kit would work in a variety of Kubernetes environments. If you want to get started with a local deployment, we recommend kind.
+To deploy kind follow the `Deploy kubernetes cluster` and `Deploy local image registry` shown in the readme in ./data_pipelines_operator  without the `docker push`.
+
+To install the service, we will push its components to the Docker regsitry and deploy it through helm.
+
+From inside `` run:
+```
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 012285273210.dkr.ecr.us-west-2.amazonaws.com
```
Comment (tozka) [inline projects/control-service/CONTRIBUTING.md:99]: Is ECR really still needed here? If it's left over let's remove it.

**Review by tozka — APPROVED** [review]
LGMT

**Comment by tozka** [comment]
If there's a ticket issue, please link it.

**Review by murphp15 — APPROVED** [review]

File: projects/control-service/CONTRIBUTING.md (hunk 1 above)
Comment (DeltaMichael) [reply projects/control-service/CONTRIBUTING.md:99]: Not sure, I haven't tried this part locally. Opened a ticket to check it and update the docs. 

https://github.com/vmware/versatile-data-kit/issues/1780

**Pull request merged**